token-metadata/program/ProgrammableNFTGuide.md

```diff
@@ -4,320 +4,262 @@
+3. The `authorization_rules` account can be easily discovered on-chain using account derivation or via the Metaplex Read API, an RPC indexing extension run by many existing RPC providers.
 
 ## Extending the `TokenStandard`
 
 A new asset class on the Token Metadata will be added to the `TokenStandard` struct:
 
 ```rust
 pub enum TokenStandard {
-  /// This is a master edition.
-	NonFungible,
-  /// A token with metadata that can also have attrributes.
-	FungibleAsset,
-	/// A token with simple metadata.
-  Fungible,
-  /// This is a limited edition.
-	NonFungibleEdition,
-  /// An NFT with customizale behaviour for lifecycle events
-  /// (e.g. transfers, updates, etc.).
-	**ProgrammableNonFungible**,
+    /// This is a master edition.
+    NonFungible,
+    /// A token with metadata that can also have attrributes.
+    FungibleAsset,
+    /// A token with simple metadata.
+    Fungible,
+    /// This is a limited edition.
+    NonFungibleEdition,
+    /// [NEW] An NFT with customizale behaviour for lifecycle events
+    /// (e.g. transfers, updates, etc.).
+    ProgrammableNonFungible,
 }
 ```
 
-When a `ProgrammableNonFungible` asset is created, it will have a `RuleSet` associated. These rules are then validated at each lifecycle action and the action is only performed if the validation succeeds. Since these rules are customizable, the tokens have a *programmable* behavior*.*
+When a `ProgrammableNonFungible` asset is created, it can have a `RuleSet` associated. These rules are then validated at each lifecycle action and the action is only performed if the validation succeeds. Since these rules are customizable, the tokens have a *programmable* behavior.
 
 ## Unified instructions
 
-To interact with the new asset class, a new unified set of instruction will be added to Token Metadata. It is important to note that current instructions will continue to work using the existing token standard – they will be required for interacting with `ProgrammableNonFungible` assets. At the same time, the new instructions will support all asset classes so all interaction can happen via an unified interface regardless of the asset class.
+To interact with the new asset class, a new set of instruction will be added to Token Metadata. It is important to note that current instructions will continue to work using the existing token standard – they will be required for interacting with `ProgrammableNonFungible` assets. At the same time, the **new instructions will support all asset classes** so all interaction can happen via an unified interface regardless of the asset class.
 
 Token Metadata instruction will be expanded to include:
 
 ```rust
 pub enum MetadataInstruction {
     ..,
-		// Mint a token with added metadata
+    // Create the metadata and associated accounts
+    Create(CreateArgs),
+    // Closes the accounts of an asset
+    Burn(BurnArgs),
```

Comment:
Looks like this list was ordered to be alphabetical, but if that's the case, `Burn` should come before `Create`.

Reply:
Done.